### PR TITLE
Revisions for clarity and precision

### PR DESCRIPTION
**Pull request by smart-alek**
Assorted cleanings and revisions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,119 +1,80 @@
...
-This is the public repository of <https://circleci.com/docs/>, a static website 
-generated by [Jekyll](https://jekyllrb.com/). If you find any errors or have 
-documentation requests, please feel free to contribute by following the instructions below. 
-Otherwise, you can always open an 
-[Issue](https://github.com/circleci/circleci-docs/issues) on this project.
+This is the public repository for <https://circleci.com/docs/>, a static website generated by [Jekyll](https://jekyllrb.com/). If you find any errors or have requests, you can contribute by following the instructions below.
 
-## Work on CircleCI Docs Locally
-There are two good ways to run a development server locally:
+If you can't contribute but still want to report a problem, open an [Issue](https://github.com/circleci/circleci-docs/issues) on this project. If you have a question or need help debugging a problem, **do not submit an issue**. Instead, please head to [CircleCI Discuss](https://discuss.circleci.com/), and our support team will help you out.
```
Comment (felicianotech) [inline README.md]: THANK YOU.

File: README.md [hunk 2]
```diff
@@ -1,119 +1,80 @@
...
-- Vagrant: [download directly](https://www.vagrantup.com/downloads.html), use apt-get (`sudo apt-get install vagrant`), or use brew (`brew cask install vagrant`).
-- VirtualBox: [download directly](https://www.virtualbox.org/wiki/Downloads), use apt-get (`sudo apt-get install virtualbox`), or use brew (`brew cask install virtualbox`). Best to use version 5.0+. Another Vagrant provider (such as Docker) could be used instead, but VirtualBox is the default.
+- Vagrant: [download directly](https://www.vagrantup.com/downloads.html), `brew cask install vagrant`, or `sudo apt-get install vagrant`
+- VirtualBox: [download directly](https://www.virtualbox.org/wiki/Downloads), `brew cask install virtualbox`, or `sudo apt-get install virtualbox`
 
 #### First Run
-To get a local copy of <circleci.com/docs/> up and running, you can run the 
-following commands. NOTE: The first time you run `./jctl start`, Vagrant will provision the entire VM for 
-you based on what's in `bootstrap.sh`. It'll then run Jekyll for you. The whole process can take a few minutes, but it's a one-time deal.
+To get a local copy of <circleci.com/docs/>, run the following commands:
```
Comment (felicianotech) [inline README.md]: If we're linking it, we should HTTPS link it.

File: README.md [hunk 3]
```diff
@@ -1,119 +1,80 @@
...
 This is a Bash wrapper script to talk to Jekyll & Vagrant.
 
-- start: Starts Jekyll. If Vagrant isn't running, starts Vagrant as well.
-- rebuild: Tells Jekyll to rebuild the site.
-- stop: Shuts down the entire VM (`vagrant halt`), including Jekyll.
-- restart: Restarts the Vagrant machine. Basically an alias of stop then start.
+- `start`: starts Jekyll; will also start Vagrant, if not already running
+- `rebuild`: rebuilds the site
+- `stop`: shuts down entire VM (including Jekyll)
+- `restart`: `jekyll stop && jekyll start`
```
Comment (felicianotech) [inline README.md]: If we're writing the command out, then it should be `./jctl stop && ./jctl start`.

File: README.md (hunk 1 above)
Comment (smart-alek) [inline README.md]: hahaha

File: README.md (hunk 3 above)
Comment (smart-alek) [inline README.md]: Ah, good catch, thanks.

**Comment by smart-alek** [comment]
@felicianotech addressed your comments

**Pull request merged**